src/lib.rs

```diff
@@ -93,12 +95,13 @@
 //! ```
 //! use metrics_exporter_statsd::StatsdBuilder;
 //!
-//! StatsdBuilder::from("127.0.0.1", 8125)
+//! let recorder = StatsdBuilder::from("127.0.0.1", 8125)
 //! .with_queue_size(5000)
 //! .with_buffer_size(1024)
 //! .histogram_is_distribution()
-//! .install(Some("prefix"))
-//! .unwrap_or_else(|err| println!("yikes could not configure statsd {:?}", err))
+//! .build(Some("prefix")).expect("Could not create StatsdRecorder");
```

Comment:
This is different whitespace from the other example.

```suggestion
//! .build(Some("prefix"))
//! .expect("Could not create StatsdRecorder");
```